src/components/Dropdown/Dropdown.tsx

```diff
@@ -89,6 +89,26 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
     } else if (e.key === 'ArrowUp') {
       updateFocusedItem(focusedIndex - 1);
     }
+    if (e.key === 'Tab' && !e.shiftKey) {
+      if (items.length !== 0) {
+        if (focusedIndex >= items.length-1) {
```

Comment:
nit but spaces around the minus and plus signs

Reply:
got it